File: ts/src/alphabet_character.ts
const alphabet_character = (word: string) => {
  let result = word.split("");
  for (let i = 0; i < result.length; i++) {
    for (let j = 0; j < result.length - i - 1; j++) {
      if (result[j] > result[j + 1]) {
        let temp = result[j];
        result[j] = result[j + 1];
        result[j + 1] = temp;
      }
    }
  }
  console.log(result.join(""));
};

alphabet_character("lamborgini"); // abgiilmnor
alphabet_character("hacker"); // acehkr

File: rust/src/alphabet_character.rs
pub fn alphabet_character (word: &str) {
	let mut result: Vec<char> = word.chars().collect();
	let length = result.len();
	for i in 0..length {
		for j in 0..length - i -1 {
			if result[j] > result [j + 1] {
				result.swap(j, j + 1);
			}
		}
	}
	println!("{:?}", result.into_iter().collect());
}